backend/db/ddl-19-apirun_groups.jinja.sql

```diff
@@ -0,0 +1,50 @@
+-- View: apiruns_grouped ------------------------------------------------------------------------------------
+DROP TABLE IF EXISTS public.api_runs CASCADE;
+
+CREATE TABLE IF NOT EXISTS public.api_runs (
```

Comment:
New table. I see!

Reply:
Yeah...added that table and also a view based on it called `apiruns_grouped`